insights/combiners/modinfo.py

```diff
@@ -1,14 +1,20 @@
 """
-ModInfo
```

Comment:
I'm curious why you're removing the Modinfo documentation? The code is still there even though it's deprecated.